pocs/camera/fli.py

```diff
@@ -0,0 +1,236 @@
+import time
+import os
+from threading import Event
+from threading import Timer
+from threading import Lock
+
+import numpy as np
+
+from astropy import units as u
+from astropy.io import fits
+
+from pocs.camera.camera import AbstractCamera
+from pocs.camera import libfli
+from pocs.utils.images import fits as fits_utils
+
+
+class Camera(AbstractCamera):
+
+    def __init__(self,
+                 name='FLI Camera',
+                 set_point=None,
+                 filter_type='M',
+                 *args, **kwargs):
+        kwargs['readout_time'] = 1.0
+        kwargs['file_extension'] = 'fits'
+        super().__init__(name, *args, **kwargs)
+
+        # Create a Lock that will be used to prevent overlapping exposures.
+        self._exposure_lock = Lock()
+
+        # Create an instance of the FLI Driver interface
+        self._FLIDriver = libfli.FLIDriver()
+        self.connect()
+
+        self.filter_type = filter_type
+
+        if self.is_connected:
+            if set_point is None:
+                # FLI *always* thermoregulates, all you can do is set a high set point
+                set_point = 25 * u.Celsius
```

Comment:
Should this come from a config somewhere just in case? I see it repeated below as well.

Reply:
What I'm doing here is using a set point of 25C as a stand in for turning the cooling off, because you can't actually disable the cooling on the FLI cameras.  I suppose we could (should?) make the 'no cooling' set point an (optional) parameter in the config file.

Reply:
The second use of this was a hang over from the way the SBIG class worked, where in order to disable the cooling you set the set point to `None`. I'd already changed that counter intuitive behaviour of the SBIG class so the corresponding bit from FLI too.  Now just have 25C in there as a simple default value for the FLI camera set point.